flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -334,6 +317,50 @@ void PartitionSpecCommaList(SqlNodeList list) :
     <RPAREN>
 }
 
+/**
+* Parses a create view or replace existing view statement.
+*   CREATE [OR REPLACE] VIEW view_name [ (field1, field2 ...) ] AS select_statement
+*/
+SqlCreate SqlCreateView(Span s, boolean replace) : {
+    boolean replaceView = false;
+    SqlIdentifier viewName;
+    SqlCharStringLiteral comment = null;
+    SqlNode query;
+    SqlNodeList fieldList = SqlNodeList.EMPTY;
+}
+{
+    [ <OR> <REPLACE> { replaceView = true; } ]
```

Comment:
We can remove this, the parameter `replace` already do this.
`SqlCreateTable` is the same.

Reply:
Yep, thanks.